Assets/Scripts/Tank/TankController.cs

```diff
@@ -86,7 +91,9 @@ private void OnTriggerEnter(Collider other)
 
             if (health <= 0)
             {
+                TankService.Instance.DestroyTankOrBullet(tankExplosionParticle, tankExplosionAudio);
```

Comment:
Use coroutines to destroy player tank after the explosion particle system finishes playing